packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -403,6 +412,10 @@ class _AppBarState extends State<AppBar> {
     IconThemeData appBarIconTheme = widget.iconTheme
```

Comment:
I feel like `appBarIconTheme` seems inspecific now that we have another icontheme.

Reply:
Renamed it to `overallIconTheme` and clarified in comment below that actionsIconTheme falls back to this when not set.